src/solvers/dg/2d/dg.jl

```diff
@@ -1522,18 +1502,11 @@ end
 
 
 # Prolong solution to mortars (select correct method based on mortar type)
-prolong2mortars!(dg::Dg2D) = prolong2mortars!(dg, dg.mortar_type, dg.thread_cache)
+prolong2mortars!(dg::Dg2D) = prolong2mortars!(dg, dg.mortar_type)
 
 # Prolong solution to mortars (l2mortar version)
-function prolong2mortars!(dg::Dg2D, ::Val{:l2}, thread_cache)
-  equation = equations(dg)
-
-  # Local storage for interface data of large element
-  # u_large_threaded = [zeros(nvariables(dg), nnodes(dg)) for _ in 1:Threads.nthreads()]
-  @unpack u_large_threaded = thread_cache
-
+function prolong2mortars!(dg::Dg2D, mortar_type::Val{:l2})
   Threads.@threads for m in 1:dg.n_l2mortars
-    u_large = u_large_threaded[Threads.threadid()]
```

Comment:
Where did all our caches go? This is crazy!!! :scream: :scream_cat:

Reply:
We\re now operating directly on views.

Reply:
Is this possible due the optimizations in 1.5 that allow views to be stack allocated even though they point to heap memory?

Reply:
Might be the case, yes.